lib/mbimport.js

```diff
@@ -189,20 +189,24 @@ var MBImport = (function () {
         appendParameter(parameters, 'annotation', release.annotation);
 
         // Label + catnos
-        for (let i = 0; i < release.labels.length; i++) {
-            let label = release.labels[i];
-            appendParameter(parameters, `labels.${i}.name`, label.name);
-            appendParameter(parameters, `labels.${i}.mbid`, label.mbid);
-            if (label.catno != 'none') {
-                appendParameter(parameters, `labels.${i}.catalog_number`, label.catno);
+        if (release.labels && Array.isArray(release.labels)) {
+            for (let i = 0; i < release.labels.length; i++) {
+                let label = release.labels[i];
+                appendParameter(parameters, `labels.${i}.name`, label.name);
+                appendParameter(parameters, `labels.${i}.mbid`, label.mbid);
+                if (label.catno != 'none') {
+                    appendParameter(parameters, `labels.${i}.catalog_number`, label.catno);
+                }
             }
         }
 
         // URLs
-        for (let i = 0; i < release.urls.length; i++) {
-            let url = release.urls[i];
-            appendParameter(parameters, `urls.${i}.url`, url.url);
-            appendParameter(parameters, `urls.${i}.link_type`, url.link_type);
+        if (release.urls || Array.isArray(release.urls)) {
```

Comment:
Should be AND instead of OR, but you can also use only the `isArray()` check as above:
```suggestion
        if (Array.isArray(release.urls)) {
```

Reply:
Thanks @kellnerd, I never manage to use this GitHub review successfully so I added this fix in forced pushed 137e8a55cd884a751c88d1b50ba8a32f87d77fb8. :)
This `||` error was a remaining of my previous version.
I must admit I did these fixes without actually running the library for real.
I used the previous version for my script, at the moment. 
But as soon as (if) this PR is merged, I will be able to omit `labels` and `urls` from my GnuDB.org quick and dirty import script.